### PR TITLE
added styles for new ticker footer , Footer component, added into var…

### PR DESCRIPTION
**Pull request by PietyE**
**Create news ticker slider** : ( https://trello.com/c/mRQ9hE8t/17-add-news-ticker-footer )

- added footer
- added styles for footer and news ticker content

### CODE REVIEW

File: src/components/Footer/mocks/content.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const footerText: Array<string> = [
+  'Все буде Україна!',
+  'Все буде Україна!',
+  'Все буде Україна!',
+  'Все буде Україна!',
+  'Все буде Україна!',
+  'Все буде Україна!',
+  'Все буде Україна!',
+];
```
Comment (kerrimov) [inline src/components/Footer/mocks/content.ts:9]: Do not need to create such an array. Just use for or while in the component

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <footer className="footer">
+      <div className="footer-container">
+        {footerText.map((text, index) => (
+          <Typography
+            className="footer-content"
+            variant="body1"
+            component="p"
+            key={index}
+          >
```
Comment (kerrimov) [inline src/components/Footer/Footer.tsx:19]: why do you need 2 iterators?  Use for only

Comment (PietyE) [reply src/components/Footer/Footer.tsx]: fixed

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import './Footer.scss';
+
+export const Footer = () => {
+  const footerText: Array<JSX.Element> = [];
+  for (let i = 0; i < 9; i++) {
+    footerText.push(
+      <Typography className="footer-content" variant="body1" component="p">
+        Все буде Україна!
+      </Typography>,
+    );
```
Comment (kerrimov) [inline src/components/Footer/Footer.tsx:12]: remove push

**Review by kerrimov — APPROVED** [review]

**Pull request merged**